backend/zulu/db_tools.py

```diff
@@ -1,15 +1,20 @@
 from dynaconf import settings
 from pymongo import GEOSPHERE, MongoClient
 
-
+ # a user references a story which references a point
 def points_db():
     client = MongoClient(settings.MONGO_URI).geo_data
     collection = client.points
-
     collection.create_index([("location", GEOSPHERE)])
     yield collection
-
-
+def story_db():
```

Comment:
well maybe stories need a DB for themselves, 
a goot arch will possibly be
user DB, points DB, stories DB
user ----> [bunch of points] 
point ----->[bunch of stories]